app/src/main/java/com/kaisar/xposed/godmode/service/GodModeManagerService.java

```diff
@@ -131,6 +133,11 @@ private void loadRuleData() throws IOException {
         }
     }
 
+    @Override
+    public boolean clearBaseServerDir() {
+        return mSystemServerDirUtils.baseServerDir().delete();
```

Comment:
2 problem
1.This doesn't work when sub directory exists.
2.Authentication is required and only God Manager calls are allowed, otherwise there may be risks.